server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraSessionModule.java

```diff
@@ -103,20 +103,22 @@ CassandraSessionConfiguration getCassandraSessionConfiguration(PropertiesProvide
 
     @Provides
     @Singleton
-    FetchBatchSizes getFetchBatchSizesConfiguration(PropertiesProvider propertiesProvider) {
+    BatchSizes getBatchSizesConfiguration(PropertiesProvider propertiesProvider) {
         try {
             PropertiesConfiguration configuration = propertiesProvider.getConfiguration(BATCHSIZES_FILE_NAME);
-            FetchBatchSizes fetchBatchSizes = FetchBatchSizes.builder()
-                    .metadata(configuration.getInt("metadata", FetchBatchSizes.DEFAULT_FETCH_BATCH_SIZE))
-                    .headers(configuration.getInt("headers", FetchBatchSizes.DEFAULT_FETCH_BATCH_SIZE))
-                    .body(configuration.getInt("body", FetchBatchSizes.DEFAULT_FETCH_BATCH_SIZE))
-                    .full(configuration.getInt("full", FetchBatchSizes.DEFAULT_FETCH_BATCH_SIZE))
+            BatchSizes batchSizes = BatchSizes.builder()
```

Comment:
"This has been done before" is not an argument for me sorry. I strongly think here this duplication is bad so if you do not want to change it please arg more about that